redisinsight/ui/src/pages/vector-search/components/rqe-not-available/RqeNotAvailableCard.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { StyledCard, StyledCardBody } from './ReqNotAvailableCard.styles'
+import { ModuleNotLoaded } from 'uiSrc/components'
+import { RedisDefaultModules } from 'uiSrc/slices/interfaces'
+
+export const RqeNotAvailableCard = () => {
+  return (
+    <StyledCard data-testid="rqe-not-available-card">
+      <StyledCardBody direction="column" gap="xxl">
+        <ModuleNotLoaded
```

Comment:
> RqeNotAvailableCard looks a lot like the ModuleNotFound modal... have you considered reusing this?

Very good observation, thanks for the note @dantovska. I think I have managed to reuse the existing visuals instead of defining a new interface, similar to what we already had.